Scroller/EditableWrapper.js

```diff
@@ -400,13 +409,15 @@ const EditableWrapper = (props) => {
 		// addEventListener to moveItems while scrolled
 		const scrollContentNode = scrollContentRef.current;
 		const scrollContentCenter = scrollContentNode.getBoundingClientRect().width / 2;
+		const scrollContentRight = scrollContentNode.getBoundingClientRect().right;
```

Comment:
getBoundingClientRect() need DOM accessing. and it takes time..
Let's reduce the DOM accessing.

Or I think we can store the scrollContentNode like  `mutableRef.current.centeredOffset`.
 `mutableRef.current.scrollContentRect?`.



```suggestion
		const scrollContentRect = scrollContentNode.getBoundingClientRect();
		...
```

Reply:
It's fixed as suggestion.